lizmap/www/js/filter.js

```diff
@@ -538,7 +538,7 @@ var lizLayerFilterTool = function() {
                         var cval = clist[i];
                         filter+= sep + '"' + field + '"' + " " + lk + " '%" + cval + "%' ";
                         // if postgresql use ILIKE instead for WMS filtered requests
-                        sep = ' OR ';
+                        sep = ' AND ';
```

Comment:
Is this change really part of the new feature?

Reply:
It is not. I changed branch release_3_3 to master.